cpp/meson.build

```diff
@@ -56,4 +56,10 @@ if git_description == ''
     git_description = run_command('git', 'describe', '--tags', check: false).stdout().strip()
 endif
 
+needs_integration = false
```

Comment:
This is to follow along with the CMake configuration. The option doesn't yet exist in Meson so I've hard-coded for now